### PR TITLE
Detect non-plugin content during install and error out.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Signed-off-by: Anusha Ragunathan anusha@docker.com

### CODE REVIEW

File: plugin/distribution/types.go [hunk 1]
```diff
@@ -7,6 +7,9 @@ import "errors"
 // ErrUnSupportedRegistry indicates that the registry does not support v2 protocol
 var ErrUnSupportedRegistry = errors.New("Only V2 repositories are supported for plugin distribution")
 
+// ErrUnSupportedMediaType indicates we are pulling content that's not a plugin
+var ErrUnSupportedMediaType = errors.New("Content is not a plugin")
```
Comment (aaronlehmann) [inline plugin/distribution/types.go]: nit: `Unsupported` instead of `UnSupported`.

Also, error text should be lowercase.

**Comment by aaronlehmann** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐹 with @aaronlehmann comment addressed :angel: 
@anusha-ragunathan is it for 1.12.0 ? 👼

**Comment by thaJeztah** [comment]
nit was adresses; LGTM

**Pull request merged**